source/Government.h

```diff
@@ -18,7 +18,6 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Color.h"
 #include "ExclusiveItem.h"
 #include "LocationFilter.h"
```

Comment:
Do you need to include Raiders here as well?
```suggestion
#include "LocationFilter.h"
#include "Raiders.h"
```
Also, why was `RaidFleet` removed?